tests/integration/features/integration_tests.feature

```diff
@@ -836,4 +836,60 @@ Feature: Integration tests
         @local
         Examples: Local storage
         | storage           | client encryption |
-        | local      |  with_client_encryption |
\ No newline at end of file
+        | local      |  with_client_encryption |
+
+    @22
+    Scenario Outline: Delete a backup with the presence of an incomplete backup
+        Given I have a fresh ccm cluster "<client encryption>" running named "scenario22"
+        Given I am using "<storage>" as storage provider in ccm cluster "<client encryption>"
+        When I create the "test" table in keyspace "medusa"
+        When I load 100 rows in the "medusa.test" table
+        When I run a "ccm node1 nodetool flush" command
+        When I perform a backup in "differential" mode of the node named "first_backup" with md5 checks "disabled"
+        When I load 100 rows in the "medusa.test" table
+        When I run a "ccm node1 nodetool flush" command
+        When I run a "ccm node1 nodetool compact medusa" command
+        When I perform a backup in "differential" mode of the node named "second_backup" with md5 checks "disabled"
+        When I load 100 rows in the "medusa.test" table
+        When I run a "ccm node1 nodetool flush" command
+        When I run a "ccm node1 nodetool compact medusa" command
+        When I perform a backup in "differential" mode of the node named "third_backup" with md5 checks "disabled"
+        Then I can see the backup named "first_backup" when I list the backups
+        Then I can see the backup named "second_backup" when I list the backups
+        And I delete the manifest from the backup named "second_backup"
```

Comment:
That specific action was not implemented in a generic way, so it's not working with storage backends other than local. That's why tests are failing with the Minio backend.
Would it be ok for you to duplicate it and make a generic version of it that uses our storage classes instead of just `os.remove()` ?

Reply:
I've duplicated the function, but wouldn't it make sense to just patch it?